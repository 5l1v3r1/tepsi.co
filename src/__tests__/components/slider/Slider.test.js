import React from 'react';
import { shallow } from 'enzyme';

import Slider from '../../../components/slider/Slider';

it('renders correctly', () => {
  const wrapper = shallow(<Slider />);

  expect(wrapper).toMatchSnapshot();
});
