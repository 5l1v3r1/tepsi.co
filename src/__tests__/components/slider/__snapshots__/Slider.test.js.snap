// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Slider
  className="Slider"
  dots={true}
>
  <div>
    <Slide1 />
  </div>
  <div>
    <Slide2 />
  </div>
  <div>
    <Slide3 />
  </div>
</Slider>
`;
